FrontEnd/src/App.js

```diff
@@ -17,6 +18,8 @@ import Header from './components/HeaderFooter/header/Header';
 import PrivacyPolicy from './components/PrivacyPolicyPage/privacy/PrivacyPolicyComponent';
 import ScrollToTopButton from './components/PrivacyPolicyPage/privacy/ScrollToTopButton';
 import TermsAndConditions from './components/terms-and-conditions-app/terms_conditions/TermsAndConditionsComponent';
+import { Search } from './components/SearchPage/Search';
+// import
```

Comment:
Do we need it?

Reply:
Yes, because we have the search page, which we use in search box in header.
This search page added to "Routes" in App.js

Reply:
I mean commented import. Remove commented imports at the top of file. You need to install every package manually because you added `package.json` to` .gitignore`